xbmc/network/EventServer.cpp

```diff
@@ -337,7 +337,7 @@ bool CEventServer::ExecuteNextAction()
           unsigned int actionID;
           CActionTranslator::TranslateString(actionEvent.actionName, actionID);
           CAction action(actionID, 1.0f, 0.0f, actionEvent.actionName);
-          CGUIComponent* gui = CServiceBroker::GetGUI();
+          IGUIComponent* gui = CServiceBroker::GetGUI();
           if (gui)
```

Comment:
I know this is all very much WIP and there is a long way to go with refactoring things step by step, but by browsing those changes I really wondered why the EventServer needs to know anything about the GUI? Having it trigger a sound by itself is IMO terribly wrong and should be handled by `g_application.OnAction(action)` consumers. I know it's something for a different patch, just wondered if this oddity will be gone in the long run.

Reply:
Well this oddity will go as soon as someone make it go ;) - I have no plans for the GUI component I just wanted to fix the stereoscopic regression in a way that was accepted.